upup/pkg/fi/cloudup/gce/op.go

```diff
@@ -58,7 +58,7 @@ func waitForZoneOp(client *compute.Service, op *compute.Operation) error {
 	}
 
 	return waitForOp(op, func(operationName string) (*compute.Operation, error) {
-		return client.ZoneOperations.Get(u.Project, u.Zone, operationName).Do()
+		return client.ZoneOperations.Wait(u.Project, u.Zone, operationName).Do()
```

Comment:
WAIT is better than GET

https://cloud.google.com/compute/docs/reference/rest/v1/globalOperations/wait